.github/ISSUE_TEMPLATE/flaking-test.yaml

```diff
@@ -0,0 +1,45 @@
+name: Flaking Test
+description: Report flaky tests or jobs in Kubernetes CI
+labels: kind/flake
+body:
+  - type: textarea
+    id: jobs
+    attributes:
+      label: Which jobs are flaking?
+      placeholder: |
+        Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in Kubernetes CI.
+
+        Links to go.k8s.io/triage and/or links to specific failures in spyglass are appreciated.
+
+        Please see the deflaking doc (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/flaky-tests.md) for more guidance.
```

Comment:
Similar to the enhancements tracking issue form, we'd probably need to link out in a description or create a shortlink for the long URL.

Reply:
Description works, I think.  PTAL